Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY . /go/src/github.com/flyteorg/flyteadmin/
 
 # This 'linux_compile' target should compile binaries to the /artifacts directory
 # The main entrypoint should be compiled to /artifacts/flyteadmin
-RUN make linux_compile
+RUN go mod tidy; make linux_compile
```

Comment:
Do we need this change?

Reply:
Thanks for catching this. I had made this change locally, never meant to check it in. Removed.